app/controllers/recipes_controller.rb

```diff
@@ -2,11 +2,41 @@ class RecipesController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @user = current_user
-    @recipes = @user.recipes
+    @recipes = Recipe.where(public: true).or(Recipe.where(user: current_user))
```

Comment:
[OPTIONAL]
- [ ] It would be nice to list only the logged-in user recipes on the recipes index page